docs/source/tutorials/offload_model.rst

```diff
@@ -0,0 +1,78 @@
+OffloadModel
+=============
+
+`fairscale.experimental.nn.offload.OffloadModel` API democratizes large scale distributed training by enabling
+users to train large models on limited GPU resources that would have traditionally resulted in OOM errors. 
+`OffloadModel` API wraps the given model and shards it almost equally. Each shard of the model is copied 
```

Comment:
Should we make here that it is specific for a sequential model e.g. "API wraps the given model (it needs to be defined as `Sequential`)"